defaults/main.yml

```diff
@@ -20,8 +20,8 @@ repox_path_log: /var/log
 repox_path_tomcat: /usr/share/tomcat
 
 # the user that logs in to the web interface
```

Comment:
These variables are set in the my-vars.yml file and are used for both the Repox and the Apache Proxy credentials. I made reference to this in the README of the vagrant_repox repo. 

If we do not set the vars in the defaults/mail.yml file of the Role, where would be the best place to set them?